src/utils/graphql-client/graphql-client.ts

```diff
@@ -66,19 +67,30 @@ export const createResponseLink = (config: ResponseLinkConfig): ApolloLink =>
     });
   });
 
-export const createHttpLink = (uri: string): ApolloLink =>
-  createApolloHttpLink({
-    uri,
+export const createHttpLink = (options?: HttpOptions): ApolloLink => {
+  return createApolloHttpLink(options);
+};
+
+export const switchUriLink = (uri: string): ApolloLink =>
```

Comment:
```suggestion
export const createSwitchUriLink = (uri: string): ApolloLink =>
```
давай как везде назовем